packages/code-du-travail-frontend/src/common/Answer.js

```diff
@@ -14,6 +17,13 @@ const BigError = ({ children }) => (
   </Container>
 );
 
+// manuC: Doing this feels stange, we should talk about it
```

Comment:
Prefer https://www.conventionalcommits.org similar format `<type>(<name>): <description>` and use your Github username

```
TODO(UnbearableBear): Need to do this thing on my work in progress
NOTE(UnbearableBear): This code is here because ...
FIXME(UnbearableBear): Below code is bad and this is why
FUTURE(UnbearableBear): This code is temporary and will change in the future
BUG(UnbearableBear): There is a known bug here lol
HACK(UnbearableBear): This code is unnatural but do the trick because...
```